easybuild/easyconfigs/l/libepoxy/libepoxy-1.5.2-foss-2018a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libepoxy'
+version = '1.5.2'
+
+homepage = 'http://www.linuxfromscratch.org/blfs/view/svn/x/libepoxy.html'
+description = "libepoxy is a library for handling OpenGL function pointer management."
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/anholt/libepoxy/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['1ffc3ffe0567f986ff5731d4ef3a3765e0a4f9d642fffdae303362567eecd689']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Mesa', '18.1.1'),
```

Comment:
@RvDijk We need to stick to `Mesa` 17.3.6 for dependencies of easyconfigs that use a `2018a` toolchain, for compatibility with other easyconfigs using the same toolchain...

Reply:
I see.. I'll stick to 17.3.6.